apps/code-of-conduct/app/block-user/block-user.component.scss

```diff
@@ -0,0 +1,51 @@
+:host {
+  border: 1px solid lightgray;
+  border-radius: 8px;
+  display: block;
+  max-width: 650px;
+  min-width: 500px;
+  padding: 12px;
+}
+
+.form-item {
+  margin: 8px 0;
+
+  mat-form-field {
+    width: 100%;
+  }
+}
+
+.two-column-row {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  justify-content: space-evenly;
+
+  > span {
+    flex: 1;
+  }
+}
+
+.comments {
+  mat-form-field {
+    width: 100%;
+  }
+
+  textarea {
+    height: 3.7em; // Allows three lines of text.
```

Comment:
why not `textarea[rows=3]` in the html? the `rows` attribute should do it I think